lib/home/view/widgets/search_input.dart

```diff
@@ -31,22 +28,16 @@ class SearchInputWidget extends StatelessWidget {
               fontWeight: FontWeight.bold,
             ),
           ),
-          SizedBox(
-            height: box.maxHeight * 0.02,
-          ),
+          VerticalSpacingWidget(box: box),
           Container(
             width: box.maxWidth,
-            child: FpbTextFormField(
-              key: const Key('SearchUserForPayments'),
+            child: CustomSearchInput(
```

Comment:
Please make to use the reusable textfield we have already setup, because I think the textfield (`FpbTextFormField`) forms @TikuGaelle have are not different from what you currently have @DesmondTambe .